### ISSUE TITLE
Regression test fails

### ISSUE THREAD

**Issue by meyskens** [src-d/regression-borges#9]
We're seeing a lot of tests failing in the cluster with the error `Could not execute pack`.

```
time="2018-09-21T00:22:00.241093523Z" level=error msg="Could not execute pack" borges="/cache/binaries/borges.v0.17.4-rc.1" error="exit status 1" output="unknown flag `file'
Usage:
  borges [OPTIONS] pack [pack-OPTIONS] path

Help Options:
  -h, --help                       Show this help message

[pack command options]
          --queue=                 queue name (default: borges) [$BORGES_QUEUE]
          --broker=                broker service URI (default:
                                   amqp://localhost:5672) [$BORGES_BROKER]
          --locking=               locking service configuration (default:
                                   local:) [$BORGES_LOCKING]
          --workers=               number of workers, 0 means the same number
                                   as processors (default: 1) [$BORGES_WORKERS]
          --timeout=               deadline to process a job (default: 10h)
                                   [$BORGES_TIMEOUT]
          --root-repositories-dir= path to the directory storing rooted
                                   repositories (can be local path or hdfs://)
                                   (default: /tmp/root-repositories)
                                   [$BORGES_ROOT_REPOSITORIES_DIR]
          --bucket-size=           if higher than zero, repositories are stored
                                   in bucket directories with a prefix of the
                                   given amount of characters from its root
                                   hash (default: 0) [$BORGES_BUCKET_SIZE]
          --temp-dir=              path of temporary directory to clone
                                   repositories into (default: /tmp/borges)
                                   [$BORGES_TEMP_DIR]
          --temp-dir-clean         clean temporary directory before starting
                                   [$BORGES_TEMP_DIR_CLEAN]

    Log Options:
          --log-level=             Logging level (info, debug, warning or
                                   error) (default: info) [$LOG_LEVEL]
          --log-format=            log format (text or json), defaults to text
                                   on a terminal and json otherwise
                                   [$LOG_FORMAT]
          --log-fields=            default fields for the logger, specified in
                                   json [$LOG_FIELDS]
          --log-force-format       ignore if it is running on a terminal or not
                                   [$LOG_FORCE_FORMAT]

    Profiler Options:
          --profiler-http          start HTTP profiler endpoint [$PROFILER_HTTP]
          --profiler-block-rate=   runtime.SetBlockProfileRate parameter
                                   (default: 0) [$PROFILER_BLOCK_RATE]
          --profiler-mutex-rate=   runtime.SetMutexProfileFraction parameter
                                   (default: 0) [$PROFILER_MUTEX_FRACTION]
          --profiler-endpoint=     address to bind HTTP pprof endpoint to
                                   (default: 0.0.0.0:6061) [$PROFILER_endpoint]

[pack command arguments]
  path:                            file with repositories to pack, one per line
" repo=cangallo source="go-log.v1/logger.go:37" url="git://localhost:9418/cangallo"
panic: exit status 1

goroutine 1 [running]:
main.main()
	/home/travis/gopath/src/github.com/src-d/regression-borges/cmd/regression/main.go:76 +0x5a7
```

**Comment by jfontan** [comment]
Thanks! There was a change in CLI options and environment variables for borges and we are working on fixing tooling around. The change is still not released but regression tests with master. For reference here's the PR with the changes:

https://github.com/src-d/borges/pull/338

**Comment by jfontan** [comment]
Fixed in #11
